HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/en.lproj/Localizable.strings

```diff
@@ -7,17 +7,18 @@
 */
 "ginihealth.reviewscreen.recipient.placeholder" = "Recipient";
 "ginihealth.reviewscreen.iban.placeholder" = "IBAN";
-"ginihealth.reviewscreen.amount.placeholder" = "0,00 €";
-"ginihealth.reviewscreen.usage.placeholder" = "Purpose";
+"ginihealth.reviewscreen.amount.placeholder" = "Amount";
+"ginihealth.reviewscreen.usage.placeholder" = "Reference number";
 "ginihealth.reviewscreen.next.button.title" = "Next";
+"ginihealth.reviewscreen.banking.app.button.label" = "To the banking app";
 "ginihealth.errors.no.banking.app.installed" = "Your banking app does not support this service. Have you tried ING app yet?";
-"ginihealth.reviewscreen.infobar.message" = "Please review the extracted information below";
+"ginihealth.reviewscreen.infobar.message" = "Please check the pre-filled data.";
 "ginihealth.errors.default" = "Oops something went wrong. Please try again.";
 "ginihealth.errors.failed.payment.request.creation" = "Oops something went wrong. Please try again.";
 "ginihealth.errors.failed.recipient.non.empty.check" = "Recipient is required.";
 "ginihealth.errors.failed.iban.non.empty.check" = "IBAN is required.";
 "ginihealth.errors.failed.iban.validation.check" = "IBAN is not valid";
-"ginihealth.errors.failed.amount.non.empty.check" = "Amount can't be 0";
+"ginihealth.errors.failed.amount.non.empty.check" = "Invalid.";
 "ginihealth.errors.failed.purpose.non.empty.check" = "Purpose is required.";
 "ginihealth.errors.failed.default.textfield.validation.check" = "The field is not valid";
```

Comment:
```suggestion
"ginihealth.errors.failed.default.textfield.validation.check" = "The field is not valid.";
```

Reply:
do we use "ginihealth.errors.failed.default.textfield.validation.check" = "The field is not valid.?

Reply:
no, I'll remove it and clean other 2 langs that are not used